README.md

```diff
@@ -53,10 +53,10 @@ projects:
     includes: ["**/*.graphql"]
     extensions:
       codegen:
-      - generator: graphql-binding
-        language: typescript
-        output:
-          binding: src/generated/db.ts
+        - generator: graphql-binding
```

Comment:
Is this indentation change intentional?

Reply:
Yes, I'd like to indent this line since vscode automatically indent this line when this file is saved. Probably because Prettier did it.